stellar/its.js

```diff
@@ -23,6 +39,156 @@ async function removeTrustedChain(wallet, _, chain, contractConfig, arg, options
     await broadcast(operation, wallet, chain, 'Trusted Chain Removed', options);
 }
 
+async function deployInterchainToken(wallet, _, chain, contractConfig, args, options) {
+    const contract = new Contract(contractConfig.address);
+    const caller = nativeToScVal(Address.fromString(wallet.publicKey()), { type: 'address' });
+    const minter = caller;
+    const [symbol, name, decimal, salt, initialSupply] = args;
+    const saltBytes32 = isHexString(salt) ? hexZeroPad(salt, 32) : keccak256(salt);
+
+    const operation = contract.call(
+        'deploy_interchain_token',
+        caller,
+        nativeToScVal(Buffer.from(arrayify(saltBytes32)), { type: 'bytes' }),
+        tokenMetadataToScVal(decimal, name, symbol),
+        nativeToScVal(initialSupply, { type: 'i128' }),
+        minter,
+    );
+
+    await broadcast(operation, wallet, chain, 'Interchain Token Deployed', options);
+}
+
+async function deployRemoteInterchainToken(wallet, _, chain, contractConfig, args, options) {
+    const contract = new Contract(contractConfig.address);
+    const caller = nativeToScVal(Address.fromString(wallet.publicKey()), { type: 'address' });
+    const [salt, destinationChain, gasTokenAddress, gasFeeAmount] = args;
+    const saltBytes32 = hexZeroPad(salt.startsWith('0x') ? salt : '0x' + salt, 32);
```

Comment:
salt is being treated differently compared to the previous function. Should be consistent, and use a helper to reduce duplication